### DISCUSSION TITLE
-format %c prints nothing

### DISCUSSION THREAD

**Discussion by yecril71pl** [ImageMagick/ImageMagick6#116] [Q&A]
```sh
convert rose: -set comment 'Rose is a rose is a rose is a rose' rose.png
identify -format %c rose.png
```

**Comment by dlemstra** [comment]
The comment is not stored when saving the image as a PNG file and that is why you don't get the comment when reading the image after you have saved it.

**Reply by yecril71pl** [reply]
This code sample is taken from the official documentation.

**Reply by dlemstra** [reply]
It looks like our documentation contains a mistake then. The example has been updated to use `jpg` instead of `png`.

**Reply by yecril71pl** [reply]
The comment is stored in PNG and can be retrieved as `%[comment]`.
